src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -206,6 +208,8 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
         return <NotFoundPage />;
     }
 
+    const shouldShowCardsSection = (policy?.areExpensifyCardsEnabled && paymentAccountID) ?? policy?.areCompanyCardsEnabled;
```

Comment:
@koko57 while reviewing this PR, I see a bug here

<img width="1202" alt="Screenshot 2024-10-14 at 17 19 22" src="https://github.com/user-attachments/assets/62f7803e-eb9f-4d94-8007-a68ba976e07f">

"0" is displayed in the end of WorkspaceMemberDetailsPage


```suggestion
    const shouldShowCardsSection = (policy?.areExpensifyCardsEnabled && !!paymentAccountID) ?? policy?.areCompanyCardsEnabled;
```